pilot/pkg/bootstrap/server.go

```diff
@@ -803,6 +803,7 @@ func (s *Server) waitForCacheSync(stop <-chan struct{}) bool {
 	// condition where we are marked ready prior to updating the push context, leading to incomplete
 	// pushes.
 	if !cache.WaitForCacheSync(stop, func() bool {
+		log.Errorf("====waitForCacheSync: committed %d, expected %d", s.XDSServer.CommittedUpdates.Load(), expected)
```

Comment:
should either remove entirely or drop the level and `===`

Reply:
Yes, see  #33247, once that is merged, i will remove the commits in front